maestro-cli/src/main/java/maestro/cli/cloud/CloudInteractor.kt

```diff
@@ -69,6 +68,7 @@ class CloudInteractor(
     ): Int {
         if (appBinaryId == null && appFile == null) throw CliError("Missing required parameter for option '--app-file' or '--app-binary-id'")
         if (!flowFile.exists()) throw CliError("File does not exist: ${flowFile.absolutePath}")
+        if (appFile?.toPath()?.startsWith(flowFile.toPath().absolute())?.not() == true) throw CliError("App file does not exist: ${flowFile.absolutePath}")
```

Comment:
Let's also print the relative path, I can image in might help in debugging. No strong opinions though.

Reply:
Cool. Your call👍